src/pages/projectSettings/tabs/EventLogTab/EventLogTab.tsx

```diff
@@ -38,11 +48,108 @@ export const EventLogTab: React.FC = () => {
     },
   });
 
+  type logEntry = {
+    timestamp: Date;
+    user: string;
+    before?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+    after?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+  };
+
+  const eventData = isRepo
+    ? repoEventData?.repoEvents?.eventLogEntries
+    : projectEventData?.projectEvents?.eventLogEntries;
+
   return (
-    <>
-      <div>This is the event log page for {identifier}</div>
-      <div> {JSON.stringify(projectEventData)}</div>
-      <div> {JSON.stringify(repoEventData)}</div>
-    </>
+    <div>
+      <div data-cy="event-log">
+        {((eventData as logEntry[]) || []).map(
+          ({ user, timestamp, before, after }) => (
+            /* @ts-expect-error */
+            <EventLogCard key={`event_log_${timestamp}`}>
+              <EventLogHeader user={user} timestamp={timestamp} />
+              <Table
+                data={getEventDiffLines(before, after)}
+                columns={[
+                  <TableHeader
+                    key="key"
+                    label={<span data-cy="key">Property</span>}
```

Comment:
Are these spans necessary? Does the default table styling work?

Reply:
we need it for the data-cy